build failed on package minicom

```
checking whether we are cross compiling... yes
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether mips-openwrt-linux-musl-gcc accepts -g... yes
checking for mips-openwrt-linux-musl-gcc option to accept ISO C89... none needed
checking whether mips-openwrt-linux-musl-gcc understands -c and -o together... yes
checking dependency style of mips-openwrt-linux-musl-gcc... gcc3
checking how to run the C preprocessor... mips-openwrt-linux-musl-gcc -E
checking for grep that handles long lines and -e... /home/runner/work/build-release/build-release/x-wrt/staging_dir/host/bin/grep
checking for egrep... /home/runner/work/build-release/build-release/x-wrt/staging_dir/host/bin/grep -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking for mips-openwrt-linux-gcc... (cached) mips-openwrt-linux-musl-gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether mips-openwrt-linux-musl-gcc accepts -g... (cached) yes
checking for mips-openwrt-linux-musl-gcc option to accept ISO C89... (cached) none needed
checking whether mips-openwrt-linux-musl-gcc understands -c and -o together... (cached) yes
checking dependency style of mips-openwrt-linux-musl-gcc... (cached) gcc3
checking for mips-openwrt-linux-ar... mips-openwrt-linux-musl-gcc-ar
checking the archiver (mips-openwrt-linux-musl-gcc-ar) interface... ar
checking for mips-openwrt-linux-ranlib... mips-openwrt-linux-musl-gcc-ranlib
checking for library containing strerror... none required
checking build system type... x86_64-pc-linux-gnu
checking host system type... mips-openwrt-linux-gnu
checking for ld used by GCC... mips-openwrt-linux-musl-ld
checking if the linker (mips-openwrt-linux-musl-ld) is GNU ld... yes
checking for shared library run path origin... done
checking for iconv... yes
checking for working iconv... guessing yes
checking how to link with libiconv... -liconv
checking pkg-config is at least version 0.9.0... yes
checking for LOCKDEV... no
./configure: line 6725: syntax error near unexpected token `external'
./configure: line 6725: `AM_GNU_GETTEXT(external)'
Makefile:47: recipe for target '/home/runner/work/build-release/build-release/x-wrt/build_dir/target-mips_24kc_musl/minicom-2.8/.configured_a17fb5ef857664f03cd0ce37cc5ea591' failed
make[3]: *** [/home/runner/work/build-release/build-release/x-wrt/build_dir/target-mips_24kc_musl/minicom-2.8/.configured_a17fb5ef857664f03cd0ce37cc5ea591] Error 2
make[3]: Leaving directory '/home/runner/work/build-release/build-release/x-wrt/feeds/packages/utils/minicom'
time: package/feeds/packages/minicom/compile#3.60#0.48#5.89
    ERROR: package/feeds/packages/minicom failed to build.
package/Makefile:114: recipe for target 'package/feeds/packages/minicom/compile' failed
make[2]: *** [package/feeds/packages/minicom/compile] Error 1
make[2]: Leaving directory '/home/runner/work/build-release/build-release/x-wrt'
package/Makefile:110: recipe for target '/home/runner/work/build-release/build-release/x-wrt/staging_dir/target-mips_24kc_musl/stamp/.package_compile' failed
make[1]: *** [/home/runner/work/build-release/build-release/x-wrt/staging_dir/target-mips_24kc_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/runner/work/build-release/build-release/x-wrt'
/home/runner/work/build-release/build-release/x-wrt/include/toplevel.mk:228: recipe for target 'world' failed
make: *** [world] Error 2

```

--- a/utils/minicom/Makefile
+++ b/utils/minicom/Makefile
@@ -20,8 +20,6 @@ PKG_LICENSE:=GPL-2.0-or-later
 PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:minicom:minicom
 
-PKG_FIXUP:=autoreconf
-
 include $(INCLUDE_DIR)/nls.mk
 include $(INCLUDE_DIR)/package.mk
 

